tests/keras/engine/test_training.py

```diff
@@ -451,30 +501,8 @@ def test_model_with_input_feed_tensor():
                   loss_weights=loss_weights,
                   sample_weight_mode=None)
 
-    # test train_on_batch
-    out = model.train_on_batch(input_b_np,
```

Comment:
I think they're extremely similar no? The new one is parameterized, so that's one difference. The prior versions aren't 100% consistent in what they test either. In other words some tests that should exist are missed with current master. However perhaps simply adding back the deletions would help merging go more quickly.